src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -461,6 +466,25 @@ def get_quota(self, fs_id, path):
         cfs = self._cephfs_instance(fs_id)
         return cfs.get_quotas(path)
 
+    @RESTController.Resource('GET', path='/statfs')
+    @EndpointDoc("Get Cephfs statfs of the specified path",
+                 parameters={
+                     'fs_id': (str, 'File System Identifier'),
+                     'path': (str, 'File System Path'),
+                 },
+                 responses={200: GET_STATFS_SCHEMA})
+    def get_statfs(self, fs_id, path):
```

Comment:
Type annotations would be good to have for any new func introduced.

Reply:
Sorry, What do you mean about type annotations, could you give an example to illustrate?

Reply:
@dparmar18 What dose mean about this comment？Should I how to modify？

Reply:
@YiteGu type [annotations](https://docs.python.org/3/library/typing.html) are hints that are provided to show what the return type of a method is.
```suggestion
    def statfs(self, fs_id, path) -> dict:
```

Reply:
yeah, we need annotations to keep some unit tests happy (like mypy).